lesson02-variables-datatype/src/main/java/Exercise/Bai07.java

```diff
@@ -0,0 +1,45 @@
+package Exercise;
+
+import java.util.Random;
+
+public class Bai07 {
+
+	public static void main(String[] args) {
```

Comment:
did not check the code yet due to the result is wrong
Just modify the bound number rd.nextInt(5) and run few times, get the wrong result
![image](https://user-images.githubusercontent.com/47709775/74598591-74a53a00-50a6-11ea-8dbb-4bff2d4717a8.png)